runironic.sh

```diff
@@ -1,53 +1,53 @@
 #!/usr/bin/bash
 
-# mysql configuration
-PATH=$PATH:/usr/sbin/
-DATADIR="/var/lib/mysql"
-if [ ! -f "$DATADIR/ironic-initalized" ]; then
-    crudini --set /etc/my.conf mysqld max_connects 64
-    crudini --set /etc/my.conf mysqld max_heap_table_size 1M
-    crudini --set /etc/my.conf mysqld innodb_buffer_pool_size 5M
-    crudini --set /etc/my.conf mysqld innodb_log_buffer_size 512K
-    mysql_install_db --datadir=$DATADIR
-    chown -R mysql /var/log/mariadb
-    chown -R mysql $DATADIR
-    cd /usr
-    mysqld_safe --datadir=$DATADIR --user mysql &
-    sleep 1
-    ironic_password=$(echo $(date;hostname)|sha256sum |cut -c-20)
-    mysqladmin -u root password $ironic_password
-    cat > /tmp/configure-mysql.sql << EOF
-TRUNCATE mysql.user;
-CREATE USER 'ironic'@'localhost' identified by '$ironic_password';
-GRANT ALL on *.* TO 'ironic'@'localhost' WITH GRANT OPTION;
-DROP DATABASE IF EXISTS test;
-CREATE DATABASE ironic;
-FLUSH PRIVILEGES;
-EOF
-    mysql -u root -p$ironic_password -h 127.0.0.1 < /tmp/configure-mysql.sql
-    ps auxf
-    crudini --set /etc/ironic/ironic.conf database connection mysql+pymysql://ironic:$ironic_password@localhost/ironic?charset=utf8
-    ironic-dbsync --config-file /etc/ironic/ironic.conf create_schema
-    touch $DATADIR/ironic-initalized
-else
-    mysqld_safe --datadir=$DATADIR --user mysql &
-fi
-
 # Get environment settings and update ironic.conf
 IP=${IP:-"172.22.0.1"}
 HTTP_PORT=${HTTP_PORT:-"80"}
 INTERFACE=${INTERFACE:-"provisioning"}
-
-sed -i -e s/IRONIC_IP/$IP/g -e s/HTTP_PORT/$HTTP_PORT/g /etc/ironic/ironic.conf 
+MARIADB_PASSWORD=${MARIADB_PASSWORD:-"change_me"}
```

Comment:
Lets do something to make this dynamic. I'd rather create it in a shared folder across container and just skip as we go.

Reply:
I'm not sure having a shared folder just for this would be the best solution.
The idea is to override this using --env during podman run, as it shows in https://github.com/openshift-metalkube/dev-scripts/pull/241, where it's dynamically created

Reply:
Ahh! I didn't see that. I think that is perfect then!